new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItInitContainers.java

```diff
@@ -0,0 +1,443 @@
+      case "managedServers":
+        domain.getSpec().addManagedServersItem(new ManagedServer()
+            .serverName("managed-server1")
+            .serverPod(new ServerPod()
+                .addInitContainersItem(new V1Container()
+                    .addCommandItem("echo").addArgsItem("\"Hi from managed-server1\"")
+                    .name("busybox")
+                    .imagePullPolicy("IfNotPresent")
+                    .image("busybox"))));
+        break;
+      default:
+        logger.info("no match for provided case {0}", testCaseName);
+    }
+
+    // create model in image domain
+    logger.info("Creating model in image domain {0} in namespace {1} using docker image {2}",
+        domainUid, domainNamespace, miiImage);
+    createDomainAndVerify(domain, domainNamespace);
+    String adminServerPodName = domainUid + adminServerPrefix;
+    //check if pod in init state
+    checkPodInitializing(adminServerPodName,domainUid, domainNamespace);
+
+    // check that admin server pod exists in the domain namespace
+    logger.info("Checking that admin server pod {0} exists in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodExists(adminServerPodName, domainUid, domainNamespace);
+
+    // check that admin service exists in the domain namespace
+    logger.info("Checking that admin service {0} exists in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceExists(adminServerPodName, domainNamespace);
+
+    // check that admin server pod is ready
+    logger.info("Checking that admin server pod {0} is ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReady(adminServerPodName, domainUid, domainNamespace);
+
+
+    // check for managed server pods existence in the domain namespace
+    for (int i = 1; i <= replicaCount; i++) {
+      String managedServerPodName = domainUid + managedServerPrefix + i;
+      //check if pod in init state
+      checkPodInitializing(managedServerPodName,domainUid, domainNamespace);
+      // check that the managed server pod exists
+      logger.info("Checking that managed server pod {0} exists in namespace {1}",
+          managedServerPodName, domainNamespace);
+      checkPodExists(managedServerPodName, domainUid, domainNamespace);
+
+
+      // check that the managed server service exists in the domain namespace
+      logger.info("Checking that managed server service {0} exists in namespace {1}",
+          managedServerPodName, domainNamespace);
+      checkServiceExists(managedServerPodName, domainNamespace);
+
+      // check that the managed server pod is ready
+      logger.info("Checking that managed server pod {0} is ready in namespace {1}",
+          managedServerPodName, domainNamespace);
+      checkPodReady(managedServerPodName, domainUid, domainNamespace);
+
+    }
+    //check the access to managed server mbean via rest api
+    checkManagedServerConfiguration(domainNamespace, domainUid);
```

Comment:
what is the need for this check?

Reply:
just to check that domain is functioning properly, admin server can access managed servermbean, I will remove it